hail/src/main/scala/is/hail/table/Table.scala

```diff
@@ -148,6 +149,31 @@ object Table {
     ).keyBy(key, isSorted)
   }
 
+  def apply(
```

Comment:
#6416

Reply:
Not sure why this is still appearing as a diff:

https://github.com/hail-is/hail/pull/6416/files#diff-fc543ff0a72f412767e8393c89ae41eeR152